packages/create-next-app/helpers/install.ts

```diff
@@ -3,103 +3,32 @@ import { yellow } from 'picocolors'
 import spawn from 'cross-spawn'
 import type { PackageManager } from './get-pkg-manager'
 
-interface InstallArgs {
-  /**
-   * Indicate whether to install packages using npm, pnpm or Yarn.
-   */
-  packageManager: PackageManager
-  /**
-   * Indicate whether there is an active Internet connection.
-   */
-  isOnline: boolean
-  /**
-   * Indicate whether the given dependencies are devDependencies.
-   */
-  devDependencies?: boolean
-}
-
 /**
- * Spawn a package manager installation with either Yarn or NPM.
+ * Spawn a package manager installation based on user preference.
  *
  * @returns A Promise that resolves once the installation is finished.
  */
-export function install(
-  root: string,
```

Comment:
Are you sure we don't need `root` anymore? It seems like that should be passed down to where we call `<pkgManger> install`

Reply:
Oh I see there is a `process.chdir(root)` much earlier before this is called so that should work. I wonder why we had it at all then.